LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2013 Artsy, Inc.
+Copyright (c) 2017 Abe Voelker
```

Comment:
Absolutely. Sorry I thought you'd removed the original notice but you have only added yours - which is all fine of course.